Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,16 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    if official?
+      "https://github.com/Homebrew/homebrew-#{repo.strip_prefix("homebrew-")}/issues"
+    else
+      "https://github.com/#{name}/issues"
```

Comment:
I guess it's kinda weird to show github issue link for non github repo. What's more. Tap could exist without remote at all. e.g. Boxen. Another solution is we can just hide the link for custom remote, such as:
```ruby
# tap.rb
def issues_url
  "https://github.com/#{user}/homebrew-#{repo}/issues" unless custom_remote?
end

# exceptions.rb
if url = formula.tap.issues_url
  puts ... 
end
```